trace/.gitignore

```diff
@@ -0,0 +1,66 @@
+*.py[cod]
```

Comment:
Can we get rid of this file since it's the same as the .gitignore at the top level?